.prettierignore

```diff
@@ -1,9 +1,9 @@
 /coverage
 packages/*/api-docs
 packages/cli/generators/*/templates
-packages/cli/test/integration/.sandbox
+**/.sandbox
 packages/*/dist*
 examples/*/dist*
-sandbox/**/dist*
+sandbox/**/*
```

Comment:
I think `sandbox` or `/sandbox` would be sufficient - see e.g. the first line that says `/coverage`.

Just FYI, the current solution is OK too - no need to bikeshed.